block/provider/attach_volume.go

```diff
@@ -43,6 +43,13 @@ func (vpcs *VPCSession) AttachVolume(volumeAttachmentRequest provider.VolumeAtta
 	defer vpcs.Logger.Debug("Exit from AttachVolume method...")
 	defer metrics.UpdateDurationFromStart(vpcs.Logger, "AttachVolume", time.Now())
 	var err error
+
+	vpcs.Logger.Info("Checking if ContainerServiceSession... is valid")
```

Comment:
can you move this logging line inside the if condition and say that iks session is not valid